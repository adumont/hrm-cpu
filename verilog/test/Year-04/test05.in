56 6d
07 04
38 ec
45 91
01 a6
03 f6
05 92
17 a0
19 a6
21 6b
21 a5
33 82
39 eb
37 96
59 8d
34 9e
24 b1
08 f8
10 de
46 4a
44 73
36 ad
14 08
