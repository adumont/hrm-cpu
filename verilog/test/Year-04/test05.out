04
6d
91
ec
f6
a6
a0
92
6b
a6
82
a5
96
eb
9e
8d
f8
b1
4a
de
ad
73
